packages/data-api/package.json

```diff
@@ -18,6 +18,7 @@
     "test:coverage": "yarn test --coverage",
     "install-mv-refresh": "scripts/installMvRefreshModule.sh",
     "build-analytics-table": "scripts/buildAnalyticsMaterializedView.sh",
+    "full-refresh-analytics-table": "scripts/fullRefreshAnalyticsTable.sh",
```

Comment:
I had in mind that it might be nice for this script to be `refresh-analytics-table` which takes `-f` / `--full` as an argument (with fast refresh being the default). That way allowing for easily doing either fast or full refresh.

If you agree and don't mind implementing that would be awesome, but of course it's totally optional and the full refresh is the one worth putting any effort into!

Reply:
Good call! I'll work on that now